src/libraries/System.Net.HttpListener/src/System/Net/Windows/HttpListener.Windows.cs

```diff
@@ -19,6 +19,84 @@
 
 namespace System.Net
 {
+    internal sealed class HttpListenerSession
```

Comment:
Could this class be moved "downstairs" or into its own file. So that this file starts with class its named after.

Reply:
Moved